src/edgeDebugAdapter.ts

```diff
@@ -60,7 +61,47 @@ export class EdgeDebugAdapter extends CoreDebugAdapter {
         return capabilities;
     }
 
-    public launch(args: ILaunchRequestArgs): Promise<void> {
+    private async _checkPortOccupiedByEDP(address: string, port: number): Promise<void> {
+        // See if EDP is using the port, if that's the case we can still launch
+        // We see if we can hit /json/version endpoint to verify if EDP is running on the port
+        const url = `http://${address}:${port}/json/version`;
+        logger.log(`Checking if EDP is running on the port by hitting ${url}`);
+
+        const jsonResponse = await coreUtils.getURL(url, { headers: { Host: 'localhost' } })
```

Comment:
I'm not familiar with this header. What is the effect of using  { headers: { Host: 'localhost' } } ?

Reply:
I think it's just to specify the domain.